### ISSUE TITLE
7$: Improve error for missing => in arrow functions

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#220]
`=>` is missing:
```js
router.post('/room/join', async (req, res, next) {
});
```
Let's produce a good error for this code. Current error:
```
hello.js:1:49: error: expected ')' to close function call [E141]
hello.js:1:12: note: function call started here [E141]
hello.js:1:49: error: missing semicolon after statement [E027]
hello.js:2:2: error: unmatched parenthesis [E056]
```

**Comment by strager** [comment]
Implemented in commit 1da1669f40e80ba6ce15c5d5860252d957f81028.
